src/wp-includes/script-loader.php

```diff
@@ -1843,6 +1843,32 @@ function wp_just_in_time_script_localization() {
 	);
 }
 
+
+/**
+ * Prints a loader script if there is text/template registered script.
+ *
+ * When added to the DOM, this script converts any text/template scripts
+ * associated with a handle into type/javascript, and executes them.
+ */
+function wp_print_template_loader_script() {
+	$wp_scripts = wp_scripts();
+
+	if ( $wp_scripts->has_delayed_inline_script() ) {
+		$output    = <<<JS
+function wpLoadAfterScripts( handle ) {
+	let scripts = document.querySelectorAll(`[type="text/template"][data-wp-executes-after="\${handle}"]`);
+	scripts.forEach( (script) => {
+		script.setAttribute("type","text/javascript");
+		eval(script.innerHTML);
+	})
+}
+JS;
+		$type_attr = current_theme_supports( 'html5', 'script' ) ? '' : " type='text/javascript'";
+
+		printf( "<script%s id='wp-executes-after-js'>\n%s\n</script>\n", $type_attr, $output );
```

Comment:
Per above, this could take advantage of `wp_print_inline_script_tag()` to construct the tag.

Reply:
@westonruter this has been addressed in a recent commit, thank you.